app.js

```diff
@@ -19,17 +17,8 @@ app.use(require('morgan')('dev'));
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
-app.use(
-  session({
-    secret: 'authorshaven',
-    cookie: { maxAge: 60000 },
-    resave: false,
-    saveUninitialized: false
-  })
-);
-
 app.use('/api/v1/users', authRoutes);
-app.use('/api/v1/user', userRoutes);
+app.use('/api/v1/user', verifyToken, userRoutes);
```

Comment:
I like the fact that you have tried to put the `verifyToken` middleware before every user route. However, the slight challenge with this is that there are some other middlewares that may need to come before it. A simple example is the check for if the param is a number while getting/updating a user.

To this end, this middleware may need to be taken to the user routes file/module and placed in the appropriate position in the stack of middlewares.

Reply:
Please, I'm not sure I fully understand where to move the middleware in the userRoutes file.
Are the endpoints for getting all users and getting a specific user the same?

Reply:
I don't think we currently have an endpoint for getting all users nor the logic to do that.

Regarding the middleware, update your branch with all the latest changes from `develop`, then check how middlewares are used in the `userRoutes` file.